### PR TITLE
How to report a bug

### PR DESCRIPTION
**Pull request by etagwerker**
Hey @schmierkov @mauro-oto, 

I've written an article about filing issues. 

Please check it out.

Thanks!

### CODE REVIEW

File: _posts/2016-01-01-how-to-report-a-bug.markdown [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+* If you found the problematic line in the library, you could enhance the tests to cover the scenario that you are seeing. The best libraries have near 100% code coverage, so adding another scenario could be easier than you think. You don't need to find the solution, but seeing a failing spec will definitely make it easier to find a solution.
+
+### 4. Share your monkeypatch
+
+Most of us will [monkeypatch](http://devblog.avdi.org/2008/02/23/why-monkeypatching-is-destroying-ruby) our application and move on. **This sucks!**
+
+You should file the issue, so that other programmers will benefit from your wasted effort.
+
+If you monkeypatched it in a horrible way, add it to the issue as well. The project maintainer or other programmers might find it that it isn't such a horrible patch after all.
```
Comment (mauro-oto) [inline _posts/2016-01-01-how-to-report-a-bug.markdown]: @etagwerker I'd drop the first `it` in `might find it that it`.

**Comment by mauro-oto** [comment]
@etagwerker nice article! :+1:

File: _posts/2016-01-01-how-to-report-a-bug.markdown (hunk 1 above)
Comment (etagwerker) [inline _posts/2016-01-01-how-to-report-a-bug.markdown]: @mauro-oto Good point.

**Comment by etagwerker** [comment]
@mauro-oto Done reviewing your comment!

**Pull request merged**